android/common/src/main/java/com/novoda/peepz/RadioButtonWidget.java

```diff
@@ -5,15 +5,17 @@
 import android.support.design.internal.ForegroundLinearLayout;
 import android.util.AttributeSet;
 import android.view.View;
-import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.novoda.peepz.common.R;
```

Comment:
does this need R as well ? what's R2?

Reply:
daniele [< 1 minute ago]
It’s generated by ButterKnife for android library projects (like the `common` module)

Reply:
We still need R because resources are still generated in that file